File: core/src/com/rpsg/rpg/object/base/items/ItemType.java
package com.rpsg.rpg.object.base.items;

/**
 * GDX-RPG Item types
 * Register a new item type from here.
 * @author dingjibang
 *
 */
public enum ItemType {
	item("道具"),
	cooking("料理"),	
	equipment("装备"),
	metrial("材料"),	
	spellcard("符卡"),
	task("任务")
	
	;
	private String value;
	private ItemType(String value){
		this.value=value;
	}
	
	public String value() {
		return value;
	}
}

File: extension/ItemEditor/ItemEditor/Enum/ItemType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ItemEditor
{
    public enum ItemType
    {
        [EnumDisplayName("道具")]
        Item,
        [EnumDisplayName("装备")]
        Equipment,
        [EnumDisplayName("任务道具")]
        Task,
        [EnumDisplayName("符卡")]
        Spellcard,
        [EnumDisplayName("食物")]
        Cooking,
        [EnumDisplayName("素材")]
        Material
    }
}
